installTerraInVm.sh

```diff
@@ -0,0 +1,49 @@
+
+#!/bin/bash
+
+# Check if gnupg are in apt cache-if not update cache and install dependencies
+if (apt-cache show gnupg); then
+    echo 'gnupg is already in the apt-cache'
+else
+    sudo apt-get update
+    echo 'Installing gnupg and software-properties-common...'
+    sudo apt-get install -y gnupg software-properties-common
```

Comment:
Might be better to have 2 explicit checks, one for "X shoudl be installed" and one for "X should be in the apt cache" as they are 2 different things.